src/Files.App/UserControls/Widgets/DrivesWidget.xaml.cs

```diff
@@ -52,7 +53,7 @@ private async void GoToStorageSense_Click(object sender, RoutedEventArgs e)
 				button.Tag.ToString() is not string path)
 				return;
 
-			await StorageSenseHelper.OpenStorageSenseAsync(path);
+			await WindowsLaunchAppService.LaunchStorageSensePolicySettingsAsync();
```

Comment:
Should you include the path in the arguments?

Reply:
The first implementation to include path is made by Gave but I'm not sure why because Storage Sense works for only system drive and for the whole system drive not for specifically a path.

Reply:
Actually path isn't used to be passed to settings to give extra context.

Reply:
There is a thread on GitHub about this with more details, but in any event, it makes a difference for which page is opened in Settings.

Reply:
Both open wrong page.

For other than C (system) drive:
![image](https://github.com/user-attachments/assets/de4f20f1-65aa-4a02-ac53-cd245b698a63)

For thers:
![image](https://github.com/user-attachments/assets/1864289f-7146-4315-af04-c21777526a4b)

Both should open:
![image](https://github.com/user-attachments/assets/9b6b53e6-8e2a-43ea-9221-30fd70d582da)

Reply:
My implementapention opens what should be opened.

Reply:
> Both open wrong page.

It's supposed to open the Storage Page for the `C:\` drive, and the "Storage Used on Other Drives" page for other drives.

Reply:
Even so, how would you free up storage for other drives? The "Storage used on other drives" doesn't contains any free up page.
As to C, why not open Policy page directly.

Reply:
I assume this weird page navigation might come from win10 when Gave implemented

Reply:
> Even so, how would you free up storage for other drives? The "Storage used on other drives" doesn't contains any free up page.

You can access the "Free up storage" page by clicking on a drive. Ideally we'd open directly to that page, but that's not supported by the Settings app.